locales/fr/translation.json

```diff
@@ -18,8 +18,8 @@
   "{{field}} must be at most {{max}} characters long.": "{{field}} doit contenir au plus {{max}} caractères.",
   "{{field}} must not contain white spaces.": "{{field}} ne doit pas contenir d'espace.",
   "{{manualLabel}} to {{automatedLabel}} ratio": "{{manualLabel}} : {{automatedLabel}}",
-  "{{month}}-{{day}}-{{year}}": "{{month}}-{{day}}-{{year}}",
-  "{{numVisibleDays}} days in view": "{{numVisibleDays}} jours de vue",
+  "{{month}}-{{day}}-{{year}}": "{{day}}-{{month}}-{{year}}",
+  "{{numVisibleDays}} days in view": "{{numVisibleDays}} jours",
```

Comment:
il ne faut pas traduire "days in view" ?

Reply:
Je ne sais pas comment le traduire. "jours affichés" peut-etre?
![image](https://user-images.githubusercontent.com/22294614/53953741-6f88ee00-40d4-11e9-8a75-b76256b75a77.png)

Reply:
en fait non, pas "affichés" car c'est sur les tendances.